src/WinRT.Runtime/ObjectReference.cs

```diff
@@ -177,6 +177,16 @@ public virtual unsafe int TryAs<T>(Guid iid, out ObjectReference<T> objRef)
             return hr;
         }
 
+        public virtual unsafe ObjectReference<IUnknownVftbl> AsKnownPtr(IntPtr ptr)

```

Comment:
Should we rename this to AsFastAbiPtr?

Reply:
I think 'As' method suggests a cast operation when the ptr to cast to is unknown. So 'AsKnownPtr' suggests cast to this object reference but do not find the ptr as it is already known. 
So I think this is not just specific to fast abi but for any scenario where the ptr is already known or precalculated and we just need an object reference pointing to that ptr.
What do you think?

Reply:
I agree with that, but I was thinking about how to make it clear the addref impacts the objectref that this function is being called on.  Maybe it being an instance method does that.